analyzer/src/main/kotlin/curation/FilePackageCurationProvider.kt

```diff
@@ -47,7 +47,7 @@ class FilePackageCurationProvider(curationFiles: Collection<File>) : PackageCura
         }
     }
 
-    val packageCurations = run {
+    internal val packageCurations = run {
```

Comment:
Should this be taken into use directly in the analyzer command?

Reply:
Not the `packageCurations`, but the `from` function.
Added a commit to reuse the functionality.